varia/sass/components/header/_site-mobile-nav-side.scss

```diff
@@ -0,0 +1,86 @@
+// Updates the Mobile Navigation to be next to the site title.
+@include media(mobile-only) {
+	.mobile-nav-side {
+		.site-header.has-menu {
+			position: relative;
+			display: grid;
+			grid-template-columns: repeat(4, 1fr);
+			grid-gap: 2px;
+
+			& > * {
+				grid-column-start: 1;
+				grid-column-end: span 3;
+				margin: 0;
+			}
+
+			.site-title {
+				align-self: end;
+			}
+
+			.site-description {
+				align-self: start;
+			}
+		}
+
+		.site-header .main-navigation ~ * {
+			grid-column-start: 1;
+			grid-column-end: span col4-end;
+		}
+
+		.site-header #site-navigation.main-navigation {
+			grid-column-start: 4;
+			grid-column-end: col4-end;
+			grid-row-start: row1-start;
+			grid-row-end: 4;
+			margin: 0;
+			justify-self: end;
+			text-align: right;
+
+			#toggle-cart[for="woocommerce-toggle"].button,
+			#toggle-menu {
+				padding: (1.5 * $baseline-unit);
+				display: inline-block;
+				text-align: center;
+				justify-self: end;
+				font-size: #{map-deep-get($config-global, "font", "size", "xs" )};
+			}
+			#toggle-cart[for="woocommerce-toggle"] {
+				margin-top: $baseline-unit;
+			}
+			#toggle-cart[for="woocommerce-toggle"].button svg {
+				margin-top: -2px;
+			}
+
+			.menu-main-menu-container {
```

Comment:
This class is different depending on the name of the menu. I think we'll need to target it with `> div`

Reply:
This is now fixed